integration-tests/funqy-google-cloud-functions/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+quarkus.package.uber-jar=true
```

Comment:
Is uber jar required by GCP?

Reply:
Yes, you can send an uberjar or a use a single function class and GCP will build it for you.
Apparently, they also support sending a ZIP file with all the sources, it's new so I don't know if it will works for a Quarkus application.